circe-akka-serializer/src/main/resources/reference.conf

```diff
@@ -1,5 +1,19 @@
 # Default configuration
 
 org.virtuslab.ash {
+
   verbose-debug-logging = off
+
+  # settings for compression of the payload
+  compression {
+    # Compression algorithm.
+    # - off  : no compression
+    # - gzip : using common java gzip
+    algorithm = off
+
+    # If compression is enabled with the `algorithm` setting,
+    # the payload will be compressed if it's size is bigger than this value.
+    compress-larger-than = 0 KiB
```

Comment:
Hmmm regardless of algorithm being `off` by default... shouldn't the default threshold be sth more usable, like 32 KiB? How does it look like in Akka Jackson's reference.conf?

Reply:
Yeah, you're right - they have 32 KiB as default (see https://doc.akka.io/docs/akka/current/serialization-jackson.html#compression for reference). Let me change that in upcomming PR fixes commit. We'll set it to 32 KiB but leave it "off" for now. Does that make sense?

Reply:
fixed

Reply:
> Does that make sense?

Indeed, that's what I meant